qemu/tests/device_option_check.py

```diff
@@ -109,6 +110,18 @@ def run(test, params, env):
                            " Guest output is '%s'" % (params_name,
                                                       output))
 
+        if sg_vpd_cmd:
+            error_context.context("Check serial number length with command %s"
+                                  % sg_vpd_cmd, logging.info)
+            sg_vpd_cmd = utils_misc.set_winutils_letter(session, sg_vpd_cmd)
+            output = session.cmd_output(sg_vpd_cmd)
+            actual_len = len(output.splitlines()[1].split()[-1])
+            actual_len = actual_len + len(output.splitlines()[2].split()[-1])
```

Comment:
```suggestion
            actual_len = sum(len(_.split()[-1]) for _ in output.splitlines()[1:3])
```

Reply:
Updated ,and paste the test result, thanks